test/cpp/c10d/TCPStoreTest.cpp

```diff
@@ -253,94 +169,6 @@ TEST(TCPStoreTest, testWatchKeyCallbackWithPrefix) {
   testWatchKeyCallback("testPrefix");
```

Comment:
This test and the test above also need to be removed.

Reply:
fixed